regression-test/suites/nereids_rules_p0/mv/partition_mv_rewrite.groovy

```diff
@@ -371,4 +368,165 @@ suite("partition_mv_rewrite") {
     order_qt_query_16_0_after "${ttl_partition_sql}"
 
     sql """ DROP MATERIALIZED VIEW IF EXISTS ${ttl_mv_name}"""
+
+
+    // date roll up mv
+    def roll_up_mv_def_sql = """
+    select date_trunc(`l_shipdate`, 'day') as col1, l_shipdate, o_orderdate, l_partkey,
+    l_suppkey, sum(o_totalprice) as sum_total
+    from lineitem
+    left join orders on lineitem.l_orderkey = orders.o_orderkey and l_shipdate = o_orderdate
+    group by
+    col1,
+    l_shipdate,
+    o_orderdate,
+    l_partkey,
+    l_suppkey;
+    """
+
+    def roll_up_all_partition_sql = """
```

Comment:
What is the difference between roll_up_mv_def_sql and roll_up_all_partition_sql？

Reply:
one is used by mv def, the other is used by query, the sql is the same